ArticleTemplates/assets/scss/modules/_live-promo.scss

```diff
@@ -0,0 +1,69 @@
+.block--live-promo {
+    background-color: color(tone-highlight);
```

Comment:
Do we want the darker yellow in dark mode?

Reply:
Yes you're right! I'll add that